src/pl/senla/hotel/ie/file/archive/EntitiesConverter.java

```diff
@@ -12,74 +13,36 @@
 import pl.senla.hotel.utils.RoomReservationUtil;
 import pl.senla.hotel.utils.RoomUtil;
 
-import java.io.BufferedReader;
-import java.io.FileReader;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-
 import static pl.senla.hotel.constant.InputOutputConstant.*;
+import static pl.senla.hotel.constant.PropertiesConstant.*;
 
-public class ReaderWriter<T> implements ReaderWriterUniversal<T>{
+public class EntitiesConverter<T> implements Converter<T> {
```

Comment:
remove it, you don't need this anymore

Reply:
I don't all classes in package src/pl/senla/hotel/ie/file/archive/.
Deleted them all.